src/gluonts/model/naive_2/__init__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# Relative imports
+from ._estimator import Naive2Estimator
+from ._predictor import Naive2Predictor
+from ._predictor import naive_02
+
+__all__ = ["Naive2Estimator", "Naive2Predictor", "naive_02"]
```

Comment:
If renaming `naive_02` into `naive_2` is a problem because of this, then call the function `predict_naive_2`, for example